src/config.rs

```diff
@@ -1019,6 +1031,38 @@ impl Config {
     }
 }
 
+/// Window Dimensions
+///
+/// Initial window dimensions in characters
+#[derive(Debug, Deserialize)]
+pub struct Dimensions {
```

Comment:
Looks like Point has `col` and `line`. That would make it less suitable I would think